misk/src/main/kotlin/misk/client/ClientMetricsInterceptor.kt

```diff
@@ -9,14 +10,15 @@ import javax.inject.Singleton
 
 class ClientMetricsInterceptor internal constructor(
   private val actionName: String,
-  private val requestDuration: Histogram
+  private val requestDuration: PrometheusHistogram
```

Comment:
client code (code that records metrics) should not have a reference to any specific histogram impl. This should be using some interface